jscomp/bsb_exe/rescript_main.ml

```diff
@@ -132,6 +136,7 @@ let build_subcommand ~start argv argv_len =
          Always regenerate build.ninja no matter bsconfig.json is changed or \
          not" );
       ("-no-deps", unit_set_spec no_deps_mode, "*internal* Needed for watcher to build without dependencies on file change");
+      ("-warn-error", string_call (fun s -> warning_as_error := Some s), "Warning numbers and whether to turn it into error or not")
```

Comment:
I think we can make the message even clearer.
Something like this maybe, what do you think?

```suggestion
      ("-warn-error", string_call (fun s -> warning_as_error := Some s), "Warning numbers and whether to turn them into errors, e.g., \"+8+32\"")
```

Reply:
Done https://github.com/rescript-lang/rescript-compiler/pull/6717/commits/1c05d6ac43fbdb0de69e1a6e4f7a3c3a9e309d51